yarn.lock

```diff
@@ -1599,6 +1599,14 @@
     rollup-pluginutils "^2.3.3"
     slash "^2.0.0"
 
+"@ovh-ux/manager-navbar@^0.2.0-alpha.2":
```

Comment:
`yarn install` from `decfde28ade6581f031b7d618c5a2f5c3782a995` causes this following change.

Reply:
fixed on `develop` branch

Reply:
branch `bugfix/module-core` is now rebase on `develop`.